### PR TITLE
Updata fast food in Taiwan

### PR DESCRIPTION
**Pull request by Supaplextw**
fix J&G Fired Chicken wrong Wikidata QID, add Dandan Hamburger, mirror fix

### CODE REVIEW

File: data/brands/amenity/fast_food.json [hunk 1]
```diff
@@ -8760,6 +8760,7 @@
         "brand": "八方雲集",
         "brand:en": "Eight Way",
         "brand:wikidata": "Q28417381",
+        "brand:wikipedia": "zh:八方雲集",
```
Comment (LaoshuBaby) [inline data/brands/amenity/fast_food.json:8763]: @Supaplextw In yesterday‘s https://github.com/osmlab/name-suggestion-index/pull/7079#discussion_r979427614, you don't need to add `brand:wikipedia` in NSI

File: data/brands/amenity/fast_food.json [hunk 2]
```diff
@@ -9423,7 +9443,9 @@
     {
       "displayName": "美而美",
       "id": "meiandmei-65b4e8",
-      "locationSet": {"include": ["cn"]},
+      "locationSet": {"include": ["cn"],
+                    "exclude": ["tw"]
```
Comment (LaoshuBaby) [inline data/brands/amenity/fast_food.json:9447]: Currently in NSI's location confliction, geocode "cn" contain “hk”&“mo” but don't contain "tw", so this exclude maybe a little unnecessary

File: data/brands/amenity/fast_food.json [hunk 3]
```diff
@@ -9668,10 +9691,34 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "すき家",
+      "locationSet": {
+        "include": ["tw"],
+        "exclude": ["hk"]
```
Comment (LaoshuBaby) [inline data/brands/amenity/fast_food.json:9698]: 台湾本来就不含香港啊

File: data/brands/amenity/fast_food.json [hunk 4]
```diff
@@ -9730,6 +9777,26 @@
...
+        "brand:zh": "丹丹漢堡",
+        "brand:zh-Hant": "丹丹漢堡",
+        "cuisine": "burger",
+        "name": "丹丹漢堡",
+        "name:en": "Dandan Hamburger",
+        "name:zh": "丹丹漢堡",
+        "name:zh-Hant": "丹丹漢堡",
+        "takeaway": "yes"
+      }
+    },
```
Comment (LaoshuBaby) [inline data/brands/amenity/fast_food.json:9800]:
```suggestion
    }
```

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

**Review by Supaplextw — COMMENTED** [review]
Add and edit stuff according to reviewer suggestions

File: data/brands/amenity/fast_food.json [hunk 5]
```diff
@@ -9730,6 +9768,33 @@
         "name:zh-Hans": "麦当劳",
         "takeaway": "yes"
       }
+    },
```
Comment (LaoshuBaby) [inline data/brands/amenity/fast_food.json:9771]:
```suggestion
    },{
```

**Pull request merged**